ReleaseNotes/eng/Altibase_7_3_0_0_1_Release_Notes.md

```diff
@@ -0,0 +1,856 @@
+  - getURL(int)
+  - getUnicodeStream(int)
+  - updateArray(int, Array)
+  - updateRef(int, Ref)
+
+###### DatabaseMetaData
+
+SPECIFIC_NAME column has been added to the results of the getProcedures(), getProcedureColumns(), getFunctions(), and getFunctionColumns() interfaces.
+In Altibase 7.3 JDBC, SPECIFIC_NAME is implemented as follows.
+
+```java
+ProcName(FuncName) + '_' + ouid
+```
+
+###### JDBC Connection Properties
+
+- [reuse_resultset](https://github.com/ALTIBASE/Documents/blob/master/Manuals/Altibase_7.3/eng/JDBC%20User's%20Manual.md#reuse_resultset)
+  - In Altibase 7.3, the default value is 'true', which means that ResultSet objects are reused. 
+  - In Altibase 7.1, the default value is 'false', which means that ResultSet objects are not reused.
+- [lob_null_select](https://github.com/ALTIBASE/Documents/blob/master/Manuals/Altibase_7.3/eng/JDBC%20User's%20Manual.md#lob_null_select)
+  - When a LOB column's value is NULL, the JDBC connection property 'lob_null_select' has been introduced to control the behavior of getBlob() and getClob() functions. 
+  - In Altibase 7.3, the default value is 'off', which means getBlob() and getClob() functions return 'NULL'.
+  - In Altibase 7.1, the default value is 'on', which means thease functions return LOB objects.
+
+- [getprocedures_return_functions](https://github.com/ALTIBASE/Documents/blob/master/Manuals/Altibase_7.3/eng/JDBC%20User's%20Manual.md#getprocedures_return_functions)
+  - Configuration for including function information in the results of DatabaseMetaData.getProcedures() and getProcedureColumns() is provided. While the JDBC API Specification 4.2 standard excludes function information, Altibase 7.3 JDBC Driver includes it for client backward compatibility. To exclude function information in accordance with the JDBC 4.2 standard, set this property value to false.
+
+###### CLIENT_TYPE
+
+The CLIENT_TYPE of an Altibase 7.3 JDBC session is NEW_JDBC42.
+
+</br>
+
+##### Changes in SQL Results and Execution Plans
+
+- Improved SQL Performance When Using the ORDER BY Clause in Subquery Inline Views
+
+  This change has an impact on the execution plan of SQLs that utilize it, with the SORT plan node being eliminated within the SUBQUERY FILTER.
+
+- Optimization of Nested LEFT OUTER JOIN Operations
+
+  This change can result in alterations to the execution plan and potential differences in the SQL execution results for SQLs affected by it.
+
+- Modifications and Additions to Subquery Unnesting Functionality
+
+  This change can lead to alterations in the execution plans of SQLs influenced by it.
+
+</br>
+
+##### Replication Restrictions
+
+###### Replication Restrictions between Altibase 7.1 and Altibase 7.3
+
+Because the replication protocol version has been changed, DDL synchronization are not supported between Altibase 7.1 and Altibase 7.3.
+
+Because the database binary version has been changed, offline replication are not supported between Altibase 7.1 and Altibase 7.3.
```

Comment:
- 빠진 문장

두 문장 다 이중화 프로토콜 버전 세 자리가 일치해야 한다는 내용이 없습니다.. 점검의 의미로 기록합니다.

Reply:
1. Because the replication protocol version has been changed,
Because the database binary version has been changed
위 두문장을 빼도 되지 않을까요? 위에 호환성 부분에 내용이 나와 있는데, 아래와 같이 한 문장으로 써도 될것 같습니다.
DDL synchronization and offline replication is not supported between Altibase 7.1 and Altibase 7.3.
2. 호환성 관련 문장에서는 EAGER일때 DDL sync와 Offline Replication이 호환이 안된다고 나와 있는데 제약 사항에는 그런말이 없어서 LAZY일때는 DDL sync와 Offline Replication이 호환이 된다는 건지 헷갈립니다. 
3. 호환성에는 Offline Replication, 제약에는 offline replication으로 대문자 사용이 다르게 되어 있습니다.

Reply:
1. 아예 이유를 빼도 될 것 같습니다. 아니면 Replication Restriction 절의 위치를 2.2.2 Compatibility으로 옮기는 건 어떨까요? 내용 상 2.2.2 절이 더 관련성 있어 보입니다. Altibase Replication Backward Compatibility를 아예 헤딩5 수준의 절로 올리고 이 내용을 포함시키는 것도 좋을 것 같습니다. 
2.  추후 수정 시 DDL synchronization and offline replication are not supported between ~ when the running replication mode is EAGER. 로 수정하겠습니다.
3. 추후 수정시 통일하여 수정하겠습니다. 저는 offline replication으로 통일하고 싶은데 이견 있으시면 코멘트 부탁드립니다.

Reply:
offline replication이 다른 db에서도 쓰이는 일반적인 말이면 소문자로 아니면 대문자로 하는게 좋을것 같네요

Reply:
구글링해보니 Offline Replication으로 많이 쓰는것 같네요
Altibase의 하나의 기능으로써의 명칭을 지칭하는 것이므로 Offline Replication이라고 쓰는게 더 낫다는 생각입니다.

Reply:
다시 읽어봤는데.. Replication Restriction 절의 위치를 2.2.2 Compatibility으로 옮기는 건 어떨까요?  -> 이건 아니것 같고요.. SRID가 껴 있어서요,,
그래서 실장님 의견처럼 because ~~ 는 빼고, 지원안되는것만 쓰는게 좋을 것 같네요. 아래와 같이요~

###### Replication Restrictions Between Altibase 7.1 and Altibase 7.3
DDL synchronization and Offline replication is not supported between Altibase 7.1 and Altibase 7.3.

Reply:
코멘트의 내용을 반영하여 위치 변경 없이
Replication Restrictions Between Altibase 7.1 and Altibase 7.3
DDL synchronization and Offline Replication are not supported between Altibase 7.1 and Altibase 7.3 when the running replication mode is EAGER.
로 수정했습니다. 이견 없으시면 이 코멘트는 리졸브하겠습니다.

Reply:
when the running replication mode is EAGER. 이건 아니예요..

정확히는 EAGER mode replication, DDL synchronization and Offline Replication are not supported between Altibase 7.1 and Altibase 7.3. 입니다.

Reply:
제가 suggestion 으로 코멘트 달았으니깐, 이 리뷰는 resolve해도 될것 같아요~